state/migration_export.go

```diff
@@ -129,10 +135,34 @@ func (st *State) exportImpl(cfg ExportConfig) (description.Model, error) {
 		return nil, errors.Trace(err)
 	}
 
+	// We have a problem, that default cloud region is selected differently based on Cloud type.
+	// For Oracle (OCI) cloud, we define the region in the credentials, but for other clouds, as first
+	// region in the list of regions for the cloud.
+	//
+	// That situation will be changed in Juju 3.3, where we will have a single way of selecting the default
+	// region for all clouds. But for now, we need to handle this situation.
+	//
+	// The workaround is to check the cloud type and if it is OCI, we will use the region from the credentials.
+	cloud, err := dbModel.Cloud()
```

Comment:
Extract this into another method, `deriveCloudRegion`. This makes it obvious to remove in later branches.